conda_build/build.py

```diff
@@ -592,7 +592,7 @@ def test(m, move_broken=True):
     specs = ['%s %s %s' % (m.name(), m.version(), m.build_id())]
 
     # add packages listed in test/requires
-    specs += m.get_value('test/requires', [])
+    specs.extend(ms.spec for ms in m.ms_depends('run'))
 
```

Comment:
Does the comment need updating? Should test/requires not also be getting added to the specs?